test/0131/http-body.test.js

```diff
@@ -0,0 +1,83 @@
+const { linterForAepRule } = require('../utils');
+
+let linter;
+
+beforeAll(async () => {
+  linter = await linterForAepRule('0131', 'aep-131-http-body');
+  return linter;
+});
+
+test('aep-131-http-body should find errors', () => {
+  const oasDoc = {
+    openapi: '3.0.3',
+    paths: {
+      '/test1/{id}': {
+        get: {
+          requestBody: {
+            content: {
+              'application/json': {
+                schema: {
+                  type: 'string',
+                },
+              },
+            },
+          },
+        },
+      },
+    },
+  };
+  return linter.run(oasDoc).then((results) => {
+    expect(results.length).toBe(1);
+    const paths = results.map(({ path }) => path.join('.'));
+    expect(paths).toContain('paths./test1/{id}.get.requestBody');
+  });
+});
+
+test('aep-131-http-body should find no errors', () => {
+  const oasDoc = {
```

Comment:
nit: I wonder if it's possible to share some of these objects, so the lines of code can be reduced on the test cases?

seems like it should be possible to have a `schemas.js` with some shared examples.